File: apps/api/app/routers/tools.py
"""Read-only tools status API (business-safe)."""

from __future__ import annotations

from fastapi import APIRouter

from app.services.tools_status import ToolsStatusResponse, get_tools_status

router = APIRouter(prefix="/tools", tags=["tools"])


@router.get("/status", response_model=ToolsStatusResponse)
def tools_status() -> ToolsStatusResponse:
    """MCP + Commander Agent online probes. No secrets; no write ops."""
    return get_tools_status()

File: apps/web/src/types/tools.ts
/** Read-only tools status (MCP + 肉机). No API keys. */

export interface ToolProbe {
  id: string
  label: string
  ok: boolean
  detail?: string
  online?: boolean | null
}

export interface ToolsStatusResponse {
  ok: boolean
  probes: ToolProbe[]
  note?: string
}
